docs/install.rst

```diff
@@ -272,6 +272,16 @@ will look like this.
     PS> ./bin/crate
 
 
+Notes about Linux and macOS
```

Comment:
if you like my suggestion below, I would actually suggest retitling this to something like "Try CrateDB without installing" or something? then move it to the top of the document

with this, we want to catch the attention of people who are only very slightly motivated to give CrateDB a try. we won't want to bury this info this far down the document. the target audience is never going to get that far. ideally, this would be visible above the fold

I'd be happy with even more emphasis being placed on "this is a quick way to try cratedb this is NOT HOW YOU INSTALL CRATEDB"

I'm really only motivated her to hook the attention of people who might otherwise just bounce after landing on this page

Reply:
Hi Naomi,

this resonates very much with me.

As @inviridi stated at https://community.crate.io/t/installation-on-opensuse-fails/615/3:

> Not having gotten very far with it, I can only guess how well your one-line script works, but let me say that it was the key reason for me to choose crate over influx. If I spend hours learning about your product I might want to use it in my other projects as well. And then it is great if it quick to install.
>
> Just my 0.02ct.

I also believe we should bring this installation variant back in an appropriate manner.

With kind regards,
Andreas.

Reply:
6c782d49044a7 will bring in your suggestions.